pkg/reconciler/common/transformers.go

```diff
@@ -807,3 +808,44 @@ func ReplaceDeploymentArg(deploymentName, existingArg, newArg string) mf.Transfo
 		return nil
 	}
 }
+
+// replaces the namespace in serviceAccount
+func ReplaceNamespaceInServiceAccount(targetNamespace string) mf.Transformer {
+	return func(u *unstructured.Unstructured) error {
+		if u.GetKind() != "ServiceAccount" {
+			return nil
+		}
+
+		// update namespace
+		u.SetNamespace(targetNamespace)
+
+		return nil
+	}
+}
+
+// replaces the namespace in clusterRoleBinding
+func ReplaceNamespaceInClusterRoleBinding(targetNamespace string) mf.Transformer {
+	return func(u *unstructured.Unstructured) error {
+		if u.GetKind() != "ClusterRoleBinding" {
+			return nil
+		}
+
+		crb := &rbacv1.ClusterRoleBinding{}
+		err := runtime.DefaultUnstructuredConverter.FromUnstructured(u.Object, crb)
+		if err != nil {
+			return err
+		}
+
+		// update namespace
+		for index := range crb.Subjects {
+			crb.Subjects[index].Namespace = targetNamespace
+		}
+
+		obj, err := runtime.DefaultUnstructuredConverter.ToUnstructured(crb)
+		if err != nil {
+			return err
+		}
+		u.SetUnstructuredContent(obj)
+		return nil
+	}
+}
```

Comment:
@piyush-garg I have added these transformers to replace namespace in SA and ClusterRoleBinding. I could not find existing func for this. Please let me know if there is already available. If not I can continue with this and I will a unit test for theses functions

Reply:
unit test added

Reply:
there are tranformers already